Toggl.Foundation.Tests/MvvmCross/ViewModels/TimeEntriesLog/GroupIdTests.cs

```diff
@@ -0,0 +1,116 @@
+using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using Toggl.Foundation.Models.Interfaces;
+using Toggl.Foundation.MvvmCross.ViewModels.TimeEntriesLog;
+using Toggl.Foundation.Tests.Mocks;
+using Xunit;
+
+namespace Toggl.Foundation.Tests.MvvmCross.ViewModels
+{
+    public sealed class GroupIdTests
+    {
+        public sealed class TheEqualsMethod
+        {
+            [Theory]
+            [MemberData(nameof(MatchingPairs))]
+            public void Equal(IThreadSafeTimeEntry timeEntryA, IThreadSafeTimeEntry timeEntryB)
```

Comment:
Maybe the names of the test should follow our usual conventions?

```suggestion
            public void CorrectlyCalculatesEquality(IThreadSafeTimeEntry timeEntryA, IThreadSafeTimeEntry timeEntryB)
```